WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -224,10 +228,12 @@ public function is_targetted_token( $stackPtr ) {
 			}
 
 			if ( preg_match( $group['regex'], $token_content ) === 1 ) {
-				$this->process_matched_token( $stackPtr, $groupName, $token_content );
+				$skip_to = $this->process_matched_token( $stackPtr, $groupName, $token_content );
```

Comment:
We might as well sort it out now as the potential issue has been identified and might be a lot harder to debug/recall in two years time.

So to that end, I've split up the `is_targetted_token()` method and changed how the value to be returned is determined. See the new commit I've just added